examples/python-operator/op.py

```diff
@@ -1,5 +1,10 @@
 from typing import Callable
+from enum import Enum
 
+class DoraStatus(Enum):
+    OK = 0
```

Comment:
```suggestion
    CONTINUE = 0
```
Keeping the same terminology